src/components/AppToolbar.tsx

```diff
@@ -221,23 +222,27 @@ class AppToolbarInternal extends React.Component<AppToolbarInternalProps> {
           </a>
         </div>
         <div className="maputnik-toolbar__actions" role="navigation" aria-label="Toolbar">
-          <ToolbarAction wdKey="nav:open" onClick={this.props.onToggleModal.bind(this, "open")}>
+          <ToolbarAction wdKey="nav:code-editor" onClick={() => this.props.onToggleModal("codeEditor")}>
+            <MdCode />
+            <IconText>{t("Code Editor")}</IconText>
+          </ToolbarAction>
```

Comment:
Users commonly click the first action first.

Since If the code editor is empty this is not very actionable, could we either move this after the open toolbar item or hide unless the action makes sense?

Reply:
It is acceptable actionable if you paste the style into it, but yeah, I can move this forward, I don't have strong feelings for this item, although I guess we should use a menu instead of all these buttons at the top...
But that's a different story.